ui/components/app/alert-system/alert-modal/alert-modal.tsx

```diff
@@ -137,9 +140,42 @@ function AlertHeader({
 
 function BlockaidAlertDetails() {
   const t = useI18nContext();
+  const { currentConfirmation } = useConfirmContext();
+  const { securityAlertResponse } = currentConfirmation;
+
+  let copy;
+  switch (securityAlertResponse?.reason) {
```

Comment:
@SayaGT Should this description, based on the security alert reason, also be reflected in the Alert System on mobile?